lib/rubocop/cop/style/infinite_loop.rb

```diff
@@ -5,6 +5,10 @@ module Cop
     module Style
       # Use `Kernel#loop` for infinite loops.
       #
+      # This cop is marked as unsafe as if the rule does not necessarily
```

Comment:
I think the `if` here is a mistake, as I'm having hard time understanding the sentence. :-)

Reply:
Oops, good catch.